frontend/src/scenes/persons/personsLogic.test.ts

```diff
@@ -56,4 +58,49 @@ describe('personsLogic', () => {
                 .toMatchValues(router, { searchParams: {} })
         })
     })
+
+    describe('loads a person', () => {
+        initKeaTestLogic({
```

Comment:
I think we're moving to the `beforeEach(() => {logic = ...; logic.mount()})` pattern. Might be worth switching this out

Reply:
Good to know! Switching it now.